deis-logger-beta1-test/manifests/deis-logger-svc.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: deis-logger
+  namespace: deis
+  labels:
+    heritage: deis
+    app: deis-logger
+spec:
+  ports:
+  - port: 80
+    name: http
+    targetPort: http
+  - port: 514
+    name: transport
+    targetPort: 1514
+    protocol: UDP
+  selector:
+    app: deis-logger
```

Comment:
Are we putting newlines at the end of files? I prefer yes, but no strong feelings. This PR is heterogeneous though.